scipy/special/add_newdocs.py

```diff
@@ -2410,6 +2410,50 @@ def add_newdoc(place, name, doc):
     The gamma function is often referred to as the generalized
     factorial since ``z*gamma(z) = gamma(z+1)`` and ``gamma(n+1) =
     n!`` for natural number *n*.
+
+    Parameters
+    ----------
+    z : float or complex array_like
+
+    Returns
+    -------
+    float or complex
```

Comment:
Maybe `gamma : float or complex`? Or something else?

Opened https://github.com/scipy/scipy/issues/7149 to hopefully clear up how the "Returns" section should look.